### PR TITLE
Fix #369: html output size

### LINKED ISSUE #369: $$html$$ output size problem
Hi there,

When trying to run the following [IJavascript example](http://n-riesco.github.io/ijavascript/doc/graphics.ipynb.html) 

```
$$html$$ = "<div style='background-color:olive;width:50px;height:50px'></div>";
```

I get 

image

This is what happens if I add something in the div like:

```
$$html$$ = "<div style='background-color:olive;width:50px;height:50px'>test</div>";
```

image

The next `$$svg$$` example works but it looks like the canvas size is too big:

image

Cheers!

Jun

### PR DESCRIPTION
**Pull request by lgeiger**
@n-riesco You added these lines in #207. Are there any special cases in which we needed them?

### CODE REVIEW

**Comment by n-riesco** [comment]
If I remember correctly those lines are meant to deal with large `DataFrame`s; e.g.
```python
import pandas as pd
import numpy as np
pd.DataFrame(np.random.randn(100, 16), columns=list('ABCDEFGHabcdefgh'))
```

I've just tested it. With this PR the horizontal scrollbar is only visible after scrolling to the bottom:

screenshot from 2016-07-20 14-48-02

and without, the horizontal scrollbar is always visible:

screenshot from 2016-07-20 14-50-21

**Comment by lgeiger** [comment]
On my system the scrollbars are displayed correctly with this PR.

**Comment by n-riesco** [comment]
@lgeiger Do you mean that you don't need to scroll down to the bottom of the DataFrame to see the horizontal scroll bar?

**Comment by lgeiger** [comment]
> @lgeiger Do you mean that you don't need to scroll down to the bottom of the DataFrame to see the horizontal scroll bar?

Yes I see both.

**Comment by n-riesco** [comment]
I'm using Pandas 0.17.1 and the DataFrame is styled like this `<div style="max-width:1500px;overflow:auto;">`.

**Comment by lgeiger** [comment]
I tried with pandas `0.17.1` and everything works fine for me.
The DataFrame is styled like this `<div><table border="1" class="dataframe">`. The problem is caused by `overflow: auto`.

**Comment by n-riesco** [comment]
I can reproduce this issue in Ubuntu 14.04 (using Pandas 0.13.1 and IPython 4.1.2) and Ubuntu 16.04 (using Pandas 0.17.1 and IPython 2.4.1).

**Comment by lgeiger** [comment]
@n-riesco I updated the PR. Please test if it works for you.

File: styles/hydrogen.less [hunk 1]
```diff
@@ -112,7 +112,7 @@
         overflow: auto;
     }
 
-    .bubble-result-container > * {
+    .bubble-result-container > {
```
Comment (n-riesco) [inline styles/hydrogen.less]: Remove `>`

Comment (n-riesco) [inline styles/hydrogen.less]: Also remove [these lines](https://github.com/lgeiger/hydrogen/blob/9dfc7c7a94d458c8971d14e36842b1070f712e0a/styles/hydrogen.less#L124-L125)

Comment (n-riesco) [inline styles/hydrogen.less]: Tested and it works for me.

**Comment by lgeiger** [comment]
Great! Works for me as well

**Pull request merged**